clicache/integration-test2/RegionSSLTest.cs

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.IO;
+using Xunit;
+
+namespace Apache.Geode.Client.IntegrationTests
+{
+  [Trait("Category", "Integration")]
+  public class RegionSSLTest : IDisposable
+  {
+    private readonly Cache _cacheOne;
+    private readonly GeodeServer _geodeServer;
+
+    public RegionSSLTest()
+    {
+      var pathvar = Environment.GetEnvironmentVariable("PATH");
```

Comment:
I like more test coverage, but what does this test have to do with the ticket?  Maybe I am missing some relationship between SSL and optional callback parameters.

Reply:
We didn't get a callback test with SSL working in the `RegionFailingTests.cs` so we added one.

Reply:
I also fail to see why it is included in this PR.

Reply:
We encountered a test that purported to cover the callback usage and SSL, but was in the old framework and riddled with decayed code so we took the opportunity to create a simple SSL test for the `clicache` while in the vicinity...

Reply:
meh.... ok